<!-- -->
<launch>
    <node pkg="adeye"
          type="controlSwitch"
          name="control_switch"
          output="screen"
          ns="safety_channel"/>
</launch>
